player/mpv.go

```diff
@@ -314,12 +318,17 @@ func (mpv *MPV) eventHandler(eventChan chan State) {
 			eventChan <- STATE_PLAYING
 		case C.MPV_EVENT_END_FILE:
 			eventChan <- STATE_STOPPED
-		case C.MPV_EVENT_PAUSE:
-			eventChan <- STATE_PAUSED
-		case C.MPV_EVENT_UNPAUSE:
-			eventChan <- STATE_PLAYING
-		}
-	}
+        case C.MPV_EVENT_PROPERTY_CHANGE:
+            prop := (*C.mpv_event_property)(event.data)
+            if uint64(event.reply_userdata) == PROP_PAUSE {
+                if *(*int32)(prop.data) != 0 {
```

Comment:
`MPV_FORMAT_FLAG` gives an `int`, which may not be `int32`. We should cast to `C.int` here instead.